### DISCUSSION TITLE
Any interest in continuing to offer the 1.0 theme?

### DISCUSSION THREAD

**Discussion by masonlee** [sapegin/squirrelsong#5] [Q&A] [answered]
Thanks for the great themes. I'm seeing that the 2.0 theme is substantially different from 1.0 (at least for vscode typescript).  My personal preference being for 1.0, I've manually installed the 1.0 version in parallel and continue to use 1.0.  Perhaps if others also like 1.0 it could be offered in addition to 2.0 as part of the same package?  Squirrelsong Light OG ?

**Comment by sapegin** [comment]
Hey @masonlee, thanks for interest in my theme! Do you have any example where you prefer 1.0? 

It should be difficult to publish an old version separately to the VS Code marketplace, though I don’t have time to maintain it. I see two ways:

* I set up a separate repository with an old theme for VSCode, publish it to the marketplace, and give you access to the repository.
* You fork it and publish yourself.

P. S. And thanks for the coffees!! Appreciate a lot — and I’m literally sitting in a café right now and drinking coffee ☕ 

P. P. S. What’s OG by the way? ;-)

**Reply by masonlee** [reply, marked as answer]
@sapegin Specifically I prefer the theme having the red and yellow colors mixed in and having only a single font (no bold/italic).  But this is just my own preference, and it is probably only because I am so used to 1.0.  For all I know, 2.0 will be more popular.

To continue using 1.0, I simply pulled your code, rolled back to one of the last 1.0 commits, changed the name/ids to "Squirrelsong Light 1.0", built it anew, and manually installed this theme into vscode.  I haven't checked to see if I'm missing out on any other 2.0 changes that I might prefer-- I probably am.

I posted here in case there are other Squirrelsong theme users who might also want 1.0.  If so, I imagine they might find this discussion and say something.  If there is demand, maybe someone can decide to publish a fork, or your theme can also include 1.0 as an option (despite that that might complicate your marketing!).

p.s. OG is slang: https://www.dictionary.com/e/slang/og/#

**Reply by sapegin** [reply]
I'm glad your found a solution!
